data/mods/Aftershock/maps/crashing_ship.json

```diff
@@ -0,0 +1,184 @@
+        "B": "f_bunkbed",
+        "f": "f_foot_locker",
+        "b": "f_bench",
+        "s": "f_shower",
+        "T": "f_toilet",
+        "w": "f_workbench",
+        "S": "f_sink",
+        "t": "f_table",
+        "V": "f_vending_reinforced",
+        "C": "f_crate_c",
+        "1": "f_arcfurnace_empty",
+        "2": "f_hydraulic_press",
+        "3": "f_air_compressor",
+        "4": "f_drill_press",
+        "9": "f_heavy_lathe",
+        "K": "f_beverly_shear",
+        "5": "f_machinery_electronic",
+        "6": "f_machinery_light",
+        "7": "f_machinery_heavy",
+        "8": "f_machinery_old",
+        "z": "f_afs_escape_pod_seat"
+      },
+      "items": { "z": { "item": "afs_escape_pod_suppliesII", "chance": 100 } }
+    },
+    "terrain": {
+      "Y": "t_bridge",
+      "!": "t_console",
+      "#": "t_console",
+      "*": "t_metal_floor_olight",
+      "M": "t_afs_space_ship_hull_wall",
+      "D": "t_afs_space_ship_hatch_c",
+      "g": "t_door_glass_frosted_c",
+      "d": "t_door_metal_locked",
+      "a": "t_gates_mech_control"
+    },
+    "mapping": {
+      "z": { "item": "afs_escape_pod_suppliesII", "chance": 100 },
+      "L": {
+        "items": [
+          { "item": "gear_soldier_sidearm", "chance": 33 },
+          { "item": "book_military", "chance": 20 },
+          { "item": "afs_plastic_armor_g", "chance": 33 },
+          { "item": "afs_carbide_raider_armor_g", "chance": 10 },
+          { "item": "mil_armor", "chance": 10 },
+          { "item": "shirts_unisex_afs", "chance": 50 },
+          { "item": "pants_unisex_afs", "chance": 50 },
+          { "item": "shoes_unisex_afs", "chance": 50 },
+          { "item": "drugs_soldier", "chance": 25 },
+          { "item": "afs_dead_survivor", "chance": 5 }
+        ]
+      },
+      "r": {
+        "items": [
+          { "item": "afs_tools_scavenging", "chance": 60 },
+          { "item": "afs_tools_robot_maintenance", "chance": 10 },
+          { "item": "afs_tools_electric_repair", "chance": 10 }
+        ]
+      },
+      "E": {
+        "items": [
+          { "item": "fireman_cabinet", "chance": 100 },
+          { "item": "full_1st_aid", "chance": 55 },
+          { "item": "full_ifak", "chance": 55 },
+          { "item": "reactor_gear", "chance": 25 }
+        ]
+      },
+      "V": { "items": [ { "item": "afs_vending_false_meals", "chance": 100 } ] },
+      "w": { "items": [ { "item": "casings", "chance": 10 } ] }
+    },
+    "fields": { "F": { "field": "fd_fire", "intensity": 3, "age": 1 } },
+    "computers": {
+      "!": {
+        "name": "Locker Room",
+        "security": 3,
+        "options": [ { "name": "UNLOCK ENTRANCE", "action": "unlock", "security": 5 } ],
+        "failures": [ { "action": "damage" }, { "action": "shutdown" } ]
+      },
+      "#": {
+        "name": "Armory Entrance",
+        "security": 6,
+        "options": [ { "name": "UNLOCK ENTRANCE", "action": "unlock_disarm", "security": 7 } ],
```

Comment:
Is it possible to open this with a starting character?  Even if it's fully specced for hacking?

Reply:
There will be an access card into the locker room and another card for the armory near the busted escape pod.